app/assets/src/components/views/SampleView/SampleView.jsx

```diff
@@ -251,6 +283,7 @@ class SampleView extends React.Component {
             // Needs to be passed down to set the background dropdown properly.
             reportPageParams={this.props.reportPageParams}
             onTaxonClick={this.handleTaxonClick}
+            onCoverageVizClick={this.handleCoverageVizClick}
```

Comment:
we'll want to add analytics tracking here at some point... see https://github.com/chanzuckerberg/idseq-web/pull/2120